app/views/catalog/_previous_next_doc.html.erb

```diff
@@ -30,9 +32,11 @@
             <li class="email">
               <%= link_to "Email", email_catalog_path(:id => @document), {:id => 'emailLink', :data => {:ajax_modal => "trigger"}, class: "icon-email"} %>
             </li>
-            <li class="endnote">
-              <%= link_to "EndNote", catalog_path(@document, :format => 'endnote'), class: "icon-external" %>
-            </li>
+            <% if @document.voyager_record? %>
+              <li class="endnote">
+                <%= link_to "EndNote", catalog_path(@document, :format => 'endnote'), class: "icon-external" %>
+              </li>
+            <% end %>
             <li class="refworks">
               <%= link_to "RefWorks", refworks_export_url({url: polymorphic_url(url_for_document(@document), format: :refworks_marc_txt, only_path: false)}), class: "icon-external", :target => "_blank"  %>
```

Comment:
Sorry the issue wasn't clear about this, but RefWorks is MARC-based and should also be Voyager-only.